### PR TITLE
Plugin added: karmada-cli

### PR DESCRIPTION
**Pull request by fleeto**
Signed-off-by: Vincent Cui <dustise@live.com>

> [Karmada](https://github.com/karmada-io/karmada) (Kubernetes Armada) is a Kubernetes management system that enables you to run your cloud-native applications across multiple Kubernetes clusters and clouds, with no changes to your applications. By speaking Kubernetes-native APIs and providing advanced scheduling capabilities, Karmada enables truly open, multi-cloud Kubernetes.
> This cli tools controls a Kubernetes Cluster Federation.

### CODE REVIEW

**Comment by corneliusweig** [comment]
@fleeto Do you intend to address the other comments?

File: plugins/karmada.yaml [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+metadata:
+  name: karmada
+spec:
+  description: >
+    Karmada (Kubernetes Armada) is a Kubernetes management system that enables
+    you to run your cloud-native applications across multiple Kubernetes
+    clusters and clouds, with no changes to your applications. By speaking
+    Kubernetes-native APIs and providing advanced scheduling capabilities,
+    Karmada enables truly open, multi-cloud Kubernetes.
+    This cli tools controls a Kubernetes Cluster Federation.
```
[the remark this answers is not in the document]
Comment (corneliusweig) [reply plugins/karmada.yaml]: Based on your explanation, I slightly tuned the wording.

[the remark this answers is not in the document]
Comment (RainbowMango) [reply plugins/karmada.yaml]: LGTM. Thanks.

[the remark this answers is not in the document]
Comment (RainbowMango) [reply plugins/karmada.yaml]: Can we move forward now? I guess all comments have been addressed.

**Comment by corneliusweig** [comment]
I slightly tuned the description. Given that merging this plugin has already been delayed quite a bit, I'll go ahead and merge now. If my wording was wrong, please  open a PR to correct that. Thanks for your patience!

/lgtm
/approve

**Pull request merged**